Controller/CRUDController.php

```diff
@@ -1349,6 +1348,28 @@ protected function preList(Request $request)
     }
 
     /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     */
+    protected function setFormTheme(FormView $formView)
```

Comment:
This method can be useful for custom actions that extends from crud controller and used forms. We have a lot `setFormTheme` calls in our actions and for now they also not works anymore.